submissions/TatyanaMolchanova/HTML-CSS-Popup/styles.css

```diff
@@ -0,0 +1,117 @@
+.nav-link img {
+    max-width: 30px;
+}
+.nav {
+    float: right;
+}
+.nav-list {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;  
+    width: 300px;
+    list-style-type: none;
+    padding-right: 30px;
+}
+.nav-item a {
+    text-decoration: none;
+    color: gray;
+}
+
+.popup-link img {
+    max-width: 50px;
+}
+
+#myhtmlpopup {
+    display: none;
+    position: absolute;
+    top: 40px;
+    width: 300px;
+    left: -200px;
+    max-height: 250px;
+    max-height: 256px;
+    overflow: auto;
+    box-shadow:1px -2px 10px 0px rgba(50, 50, 50, 0.5)
+}
+
+#mypopup:checked ~ #myhtmlpopup {
+    display: block;
+}
+
+#mypopup {
```

Comment:
Hide checkbox in the right way [video](https://www.youtube.com/watch?v=E6kLaaQFctU&ab_channel=VadimMakeev).

Reply:
Thank you, I watched this video and changed opacity:0 on checkbox to apperance: none and it fixed my problem with focus.